pkg/ipamd/datastore/data_store.go

```diff
@@ -176,7 +182,17 @@ func NewDataStore(log logger.Logger) *DataStore {
 }
 
 // AddENI add ENI to data store
-func (ds *DataStore) AddENI(eniID string, deviceNumber int, isPrimary bool) error {
+func (ds *DataStore) AddENI(eniID string, deviceNumber int, isPrimary bool, eniConfigName string) error {
+	addr := net.IPNet{
+		IP:	net.ParseIP("0.0.0.0"),
+		Mask: net.CIDRMask(32, 32),
+	}
```

Comment:
What is this `addr` needed for? A default CIDR mask?

Reply:
This is a default mask to provide backwards compatibility for the large number of callers to AddENI. I'll add a comment to make that clear.

Reply:
Actually, this was only being used in tests at this point, I've updated them to bypass this shim function and deleted it.